docs/guides/typescript.md

```diff
@@ -44,16 +44,16 @@ mv buidler.config.js buidler.config.ts
 We also need to adapt it to explicitly import the Buidler config DSL, and use the [Buidler Runtime Environment] explicitly.
 
 For example, the sample project's config turns from this
-```js{1,6,13}
-usePlugin("@nomiclabs/buidler-truffle5");
+```js{1,5,13}
```

Comment:
first line here is highlighted in there's no changes to it from one example to the next